README.md

```diff
@@ -21,27 +21,36 @@ here and there, but I think it is working.
 The code is now it's beta phase. See the implemented column for what has
 already been implemented.
 
-Dependencies:
--------------
+## Dependencies:
+
 
 Python packages required: See the environment.yml file
 
 To use the movie saving feature: ffmpeg & xterm.
 
 Iseult should work on Windows, MacOS & Linux.
 
-To run Iseult on tigressdata or stellar vis nodes type the following:
+## Setup & Running
+
+There are two different methods to setup what you need for Iseult to run. The recommended method is to setup a python environment with conda, pyenv, etc and a conda `environment.yml` file is provided. The other method for those with access to the Stellar cluster at Princeton is to simply load the proper Anaconda module.
 ```bash
-# Initial setup, optional if your python environment has the correct version of all dependencies
+# Recommended setup
+# If on Stellar
 $ module load anaconda3/2024.6 # or the latest version
```

Comment:
The code does not work with the latest version, right? Or it does not matter if environment is used?

Reply:
It doesn't matter if an environment is used. 

Anaconda is a package manager (Conda) plus a bunch of packages commonly used for data science. Making a Conda environment uses Conda to create a new environment with whatever package versions you want.